src/components/ModalPortal.js

```diff
@@ -119,9 +119,8 @@ export default class ModalPortal extends Component {
 
   beforeOpen() {
     const { appElement, ariaHideApp, bodyOpenClassName } = this.props;
-    refCount.add(bodyOpenClassName);
     // Add body class
-    elementClass(document.body).add(bodyOpenClassName);
+    bodyClassList.add(bodyOpenClassName);
```

Comment:
I've created a [commit](https://github.com/indiesquidge/react-modal/commit/6b83e2609335720548085f5d7fc20fcc973b4d34#diff-eafaf565940b2fb71c5ed190e4f65abf) on another branch to demonstrate what this might look like.